tests/ci_tests/code_formatter/conftest.py

```diff
@@ -0,0 +1,28 @@
+"""Fixtures and configuration for code_formatter tests"""
+
+import os
+from unittest import mock
+
+import pytest
```

Comment:
Added a docstring for the function as specified in the style guide.
```suggestion
    "Provide a mock Langfuse client"
```